local_flow/intent/requirements.txt

```diff
@@ -4,6 +4,8 @@ tensorflow==2.5.0
 python-dotenv==0.13.0
 scikit-learn==0.24.1
 wandb==0.10.30
+neptune-client==0.13.3
```

Comment:
@Blaizzy here (neptune-client and neptune-tf-keras) probably you can specify minimal version but also allow newer. In this way it will be easier to maintain so:
`neptune-client>=0.13.3`

Reply:
Got it!